types/backbone/index.d.ts

```diff
@@ -241,7 +241,7 @@ declare namespace Backbone {
         * For assigning an object hash, do it like this: this.defaults = <any>{ attribute: value, ... };
         * That works only if you set it in the constructor or the initialize method.
         **/
-        defaults(): ObjectHash;
+        defaults(): Partial<T>;
```

Comment:
In principle I'm in favor, but if you go this route, you should also update the properties on lines 228-236.